docs/contributing/pages/redirects.mdx

```diff
@@ -3,31 +3,57 @@ title: Redirects
 noindex: true
 ---
 
-
 Redirects allow you to automatically redirect an incoming request path to a new destination path. When you move or rename a file, you should make sure to set up a redirect from the old path to the new path, so that the old URL still takes users to the right place.
 
 ## Add a New Redirect
 
-Redirects are configured in [`next.config.js`](https://github.com/getsentry/sentry-docs/blob/master/next.config.js) and are checked before the filesystem (including pages and `/public` files ). The `redirects` function returns an array holding redirect objects.
+There are two ways to add redirects in the Sentry docs:
+
+- [Add a New Redirect in `middleware.ts` (Recommended)](#add-a-new-redirect-in-middlewarets-recommended)
```

Comment:
@stephanie-anderson Can you confirm if we want to recommend this over vercel redirects?

Reply:
Yes, if they are one to one with no dynamic segments. This way does not count towards the redirects quota on Vercel